docs/app/pages/components/components-sections/tables/filters/filters.component.html

```diff
@@ -17,21 +17,27 @@
     <tbody>
         <tr class="clickable" *ngFor="let document of documents">
             <td>
-                <div class="listview-page-icon"></div>
+                <ux-icon
+                    name="document"
+                    size="24px"
+                    [flipHorizontal]="false"
+                    [flipVertical]="false">
+                </ux-icon>
             </td>
             <td>{{ document.name }}</td>
             <td>{{ document.author }}</td>
             <td>{{ document.date }}</td>
             <td>
-                <ux-spark theme="accent"
+                <ux-spark [theme]="'ok'"
                     [value]="document.completed"
                     barHeight="3"
                     [inlineLabel]="document.completed">
                 </ux-spark>
             </td>
             <td class="text-center vertical-center-icon">
                 <ux-icon class="text-accent"
-                    [name]="document.active ? 'checkmark' : null"
+                    [style.color]="colorThemeRGB"
```

Comment:
We should remove the text-accent class, as this is setting the colour. Its probably best that we just keep this black as it is adding unnecessary complexity to the example. Also instead of changing the `name` we should just use `*ngIf` to hide, eg:

```html
<ux-icon *ngIf="document.active" name="checkmark">
```